CassiaWindowList@klangman/files/CassiaWindowList@klangman/4.0/applet.js

```diff
@@ -389,6 +406,72 @@ function getKeyAndButtonMouseAction(mouseActionList, modifier, context, mouseBtn
    return -1;
 }
 
+function moveTitleBarToScreen(window) {
+   if (majorVersion < 5) {
```

Comment:
You can move the applet code into versioned folders to avoid having to introduce these kinds of conditionals. Take a look at other applets for an example.

Reply:
I did consider doing that, but went with this approach for now to keep the code common for a little longer. I'll give up and split off to a version 5 source tree if too much more of this ugliness starts creeping in.